<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>findLocationMatches</class>
 <widget class="QWidget" name="findLocationMatches">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1116</width>
    <height>896</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="LinearGradientPattern">
       <gradient startx="0.489000000000000" starty="0.000000000000000" endx="0.517000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
        <gradientstop position="0.000000000000000">
         <color alpha="255">
          <red>65</red>
          <green>63</green>
          <blue>180</blue>
         </color>
        </gradientstop>
        <gradientstop position="1.000000000000000">
         <color alpha="255">
          <red>170</red>
          <green>0</green>
          <blue>255</blue>
         </color>
        </gradientstop>
       </gradient>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>212</red>
        <green>212</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>85</red>
        <green>85</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>113</red>
        <green>113</green>
        <blue>170</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="LinearGradientPattern">
       <gradient startx="0.489000000000000" starty="0.000000000000000" endx="0.517000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
        <gradientstop position="0.000000000000000">
         <color alpha="255">
          <red>65</red>
          <green>63</green>
          <blue>180</blue>
         </color>
        </gradientstop>
        <gradientstop position="1.000000000000000">
         <color alpha="255">
          <red>170</red>
          <green>0</green>
          <blue>255</blue>
         </color>
        </gradientstop>
       </gradient>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="LinearGradientPattern">
       <gradient startx="0.489000000000000" starty="0.000000000000000" endx="0.517000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
        <gradientstop position="0.000000000000000">
         <color alpha="255">
          <red>65</red>
          <green>63</green>
          <blue>180</blue>
         </color>
        </gradientstop>
        <gradientstop position="1.000000000000000">
         <color alpha="255">
          <red>170</red>
          <green>0</green>
          <blue>255</blue>
         </color>
        </gradientstop>
       </gradient>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>212</red>
        <green>212</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="PlaceholderText">
      <brush brushstyle="SolidPattern">
       <color alpha="128">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="LinearGradientPattern">
       <gradient startx="0.489000000000000" starty="0.000000000000000" endx="0.517000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
        <gradientstop position="0.000000000000000">
         <color alpha="255">
          <red>65</red>
          <green>63</green>
          <blue>180</blue>
         </color>
        </gradientstop>
        <gradientstop position="1.000000000000000">
         <color alpha="255">
          <red>170</red>
          <green>0</green>
          <blue>255</blue>
         </color>
        </gradientstop>
       </gradient>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>212</red>
        <green>212</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>85</red>
        <green>85</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>113</red>
        <green>113</green>
        <blue>170</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="LinearGradientPattern">
       <gradient startx="0.489000000000000" starty="0.000000000000000" endx="0.517000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
        <gradientstop position="0.000000000000000">
         <color alpha="255">
          <red>65</red>
          <green>63</green>
          <blue>180</blue>
         </color>
        </gradientstop>
        <gradientstop position="1.000000000000000">
         <color alpha="255">
          <red>170</red>
          <green>0</green>
          <blue>255</blue>
         </color>
        </gradientstop>
       </gradient>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="LinearGradientPattern">
       <gradient startx="0.489000000000000" starty="0.000000000000000" endx="0.517000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
        <gradientstop position="0.000000000000000">
         <color alpha="255">
          <red>65</red>
          <green>63</green>
          <blue>180</blue>
         </color>
        </gradientstop>
        <gradientstop position="1.000000000000000">
         <color alpha="255">
          <red>170</red>
          <green>0</green>
          <blue>255</blue>
         </color>
        </gradientstop>
       </gradient>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>212</red>
        <green>212</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="PlaceholderText">
      <brush brushstyle="SolidPattern">
       <color alpha="128">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>85</red>
        <green>85</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="LinearGradientPattern">
       <gradient startx="0.489000000000000" starty="0.000000000000000" endx="0.517000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
        <gradientstop position="0.000000000000000">
         <color alpha="255">
          <red>65</red>
          <green>63</green>
          <blue>180</blue>
         </color>
        </gradientstop>
        <gradientstop position="1.000000000000000">
         <color alpha="255">
          <red>170</red>
          <green>0</green>
          <blue>255</blue>
         </color>
        </gradientstop>
       </gradient>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>212</red>
        <green>212</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>85</red>
        <green>85</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>113</red>
        <green>113</green>
        <blue>170</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>85</red>
        <green>85</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>85</red>
        <green>85</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="LinearGradientPattern">
       <gradient startx="0.489000000000000" starty="0.000000000000000" endx="0.517000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
        <gradientstop position="0.000000000000000">
         <color alpha="255">
          <red>65</red>
          <green>63</green>
          <blue>180</blue>
         </color>
        </gradientstop>
        <gradientstop position="1.000000000000000">
         <color alpha="255">
          <red>170</red>
          <green>0</green>
          <blue>255</blue>
         </color>
        </gradientstop>
       </gradient>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="LinearGradientPattern">
       <gradient startx="0.489000000000000" starty="0.000000000000000" endx="0.517000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
        <gradientstop position="0.000000000000000">
         <color alpha="255">
          <red>65</red>
          <green>63</green>
          <blue>180</blue>
         </color>
        </gradientstop>
        <gradientstop position="1.000000000000000">
         <color alpha="255">
          <red>170</red>
          <green>0</green>
          <blue>255</blue>
         </color>
        </gradientstop>
       </gradient>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>170</red>
        <green>170</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="PlaceholderText">
      <brush brushstyle="SolidPattern">
       <color alpha="128">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0.489, y1:0, x2:0.517, y2:1, stop:0 rgba(65, 63, 180, 255), stop:1 rgba(170, 0, 255, 255));</string>
  </property>
  <widget class="QPushButton" name="closeProgram">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>820</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;</string>
   </property>
   <property name="text">
    <string>Close program</string>
   </property>
  </widget>
  <widget class="QPushButton" name="logout">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>820</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;</string>
   </property>
   <property name="text">
    <string>Logout</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>931</width>
     <height>731</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #ededed;
color: black;
</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QTextBrowser" name="output">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>450</y>
      <width>651</width>
      <height>261</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="maxDistance">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>140</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>101</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Max distance (in km):</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>330</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Location Name:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="volumeLevel">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>90</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="InOrOut">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>190</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>290</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Longitude:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>701</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Please select your preferences for your study location:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="equipment">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>240</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Equipment</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="addressName">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>330</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="wifi">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>240</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>WiFi</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>121</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Indoor, outdoor, or either:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>151</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Volume (1 is silence, 10 is loud):</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>290</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Latitude:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="power">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>240</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Power Outlet</string>
    </property>
   </widget>
   <widget class="QPushButton" name="findMatches">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>390</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Semibold</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;</string>
    </property>
    <property name="text">
     <string>Display Matches!</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="latitude">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>290</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="longitude">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>290</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>121</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>What else are you looking for?</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>121</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Where are you right now?</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>volumeLevel</tabstop>
  <tabstop>maxDistance</tabstop>
  <tabstop>InOrOut</tabstop>
  <tabstop>wifi</tabstop>
  <tabstop>power</tabstop>
  <tabstop>equipment</tabstop>
  <tabstop>longitude</tabstop>
  <tabstop>latitude</tabstop>
  <tabstop>addressName</tabstop>
  <tabstop>findMatches</tabstop>
  <tabstop>output</tabstop>
  <tabstop>closeProgram</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>closeProgram</sender>
   <signal>clicked()</signal>
   <receiver>findLocationMatches</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>438</x>
     <y>929</y>
    </hint>
    <hint type="destinationlabel">
     <x>741</x>
     <y>893</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
